### ISSUE TITLE
After Update to the latest version KeyError: 'allinone'

### ISSUE THREAD

**Issue by frauhottelmann** [ualex73/monitor_docker#147]
Hi there,

after I updated to the latest version I am getting the following error:

```
2024-05-13 15:40:22.136 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up monitor_docker platform for sensor

Traceback (most recent call last):

  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 356, in _async_setup_platform

    await asyncio.shield(awaitable)

  File "/config/custom_components/monitor_docker/sensor.py", line 147, in async_setup_platform

    CONTAINER_MONITOR_LIST[CONTAINER_INFO_ALLINONE],

    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^

KeyError: 'allinone'
```

I haven't changed anything in my config:

```
monitor_docker:
  - name: Docker
    switchenabled: false
    monitored_conditions:
      - state
      - status
      - image
      - allinone
```

Any ideas?

**Comment by Balok-Fesarius** [comment]
Same

**Comment by matteo-martinelli** [comment]
Same for me: after the update, I have the following error: 

`Error while setting up monitor_docker platform for sensor`

... with the following specification: 

`KeyError: 'allinone'`

**Comment by ualex73** [comment]
I can reproduce the error, but I do not have a direct working solution (I can remove the error, but allinone then still does not work). I need to debug more to figure out where it goes wrong.

**Comment by MikeDeltaHH** [comment]
I removed "allinone" and it worked again... thanks!

**Comment by ualex73** [comment]
Fixed in 1.16, which I just published.
